iwa-console/packages.config

```diff
@@ -9,7 +9,7 @@
   <package id="Microsoft.Extensions.FileProviders.Physical" version="2.1.1" targetFramework="net461" />
   <package id="Microsoft.Extensions.FileSystemGlobbing" version="2.1.1" targetFramework="net461" />
   <package id="Microsoft.Extensions.Primitives" version="2.1.1" targetFramework="net461" />
-  <package id="Microsoft.Identity.Client" version="2.6.2" targetFramework="net461" />
+  <package id="Microsoft.Identity.Client" version="1.0.0-localbuild" targetFramework="net461" />
```

Comment:
The NuGet package was generated locally (and I don't know how to change its number to 3.0.0-preview

Reply:
When you build and pack, add /p:MsalClientSemVer=3.0.0-preview to the msbuild cmd line